python/mxnet/module/base_module.py

```diff
@@ -80,14 +80,14 @@ class BaseModule(object):
     """The base class of a module.
 
     A module represents a computation component. Modules are designed so that
```

Comment:
Modules are designed so that they can be thought of as a computation "machine" -> One can think of module as a computation *machine*